src/table.js

```diff
@@ -306,6 +318,42 @@ export const addRowAt = rowIndex => tr => {
   return tr;
 };
 
+// :: (cloneRowIndex: number) → (tr: Transaction) → Transaction
+// Returns a new transaction that adds a new row after `cloneRowIndex`, cloning the row attributes at `cloneRowIndex`.
+//
+// ```javascript
+// dispatch(
+//   cloneRowAt(i)(state.tr)
+// );
+// ```
+export const cloneRowAt = rowIndex => tr => {
```

Comment:
I feel like it either should be a private method and used as a helper for `addRowAt`, or it should be public API that does all of the checking:
```
if (rowIndex >= 1 && rowIndex <= map.height) {
  ...
}
```

Reply:
There is a check for that already? I have moved it higher now though.